BuildingBlocks/src/BuildingBlocks.Infrastructure/Persistence/BlobStorage/AzureStorageAccount/AzureStorageAccountServiceCollectionExtensions.cs

```diff
@@ -20,6 +23,11 @@ public static void AddAzureStorageAccount(this IServiceCollection services, Azur
         services.Configure<AzureStorageAccountOptions>(opt => opt.ConnectionString = options.ConnectionString);
         services.AddSingleton<AzureStorageAccountContainerClientFactory>();
         services.AddScoped<IBlobStorage, AzureStorageAccount>();
+        services.AddSingleton(_ => new BlobServiceClient(options.ConnectionString));
+
+        services.AddHealthChecks().AddAzureBlobStorage(
+            optionsFactory: sp => new AzureBlobStorageHealthCheckOptions { ContainerName = sp.GetRequiredService<IOptions<BlobStorageOptions>>().Value.Container }
+        );
```

Comment:
Hmm, you know what? I think I like the self-implemented health check for Google Cloud more than the one of the library for Azure, because it also checks the permissions. I think we should create a custom implementation for Azure as well. 
You should even be able to only use a single implementation. This implementation could simply use our `IBlobStorage` to perform its checks.